test/python/WMCore_t/WorkQueue_t/DataStructs_t/WorkQueueElementsSummary_t.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+    WorkQueueElement unit tests
+"""
+from __future__ import print_function
+import unittest
+from WMCore.Wrappers import JsonWrapper
+from WMCore.WorkQueue.DataStructs.WorkQueueElementsSummary import WorkQueueElementsSummary
```

Comment:
Isn't it supposed to be WorkQueueElementResult class? Or did you forget to push this new class?